pkg/containers/containers.go

```diff
@@ -20,12 +20,19 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/google/go-containerregistry/pkg/crane"
 	"github.com/google/go-containerregistry/pkg/name"
-	"github.com/google/go-containerregistry/pkg/v1/remote"
-
-	"github.com/stacklok/frizbee/pkg/constants"
+	gocrv1 "github.com/google/go-containerregistry/pkg/v1"
 )
 
+// nolint: gochecknoglobals
+var platform *gocrv1.Platform
```

Comment:
I'm nervous about introducing globals and non-reentrant to this module.  Can we extend GetDigest / GetDigestFromRef to take a `...remote.Option` slice, and then use that from `one.go`?